Application/package.json

```diff
@@ -34,14 +34,15 @@
     "dotenv": "^16.4.5",
     "eslint-config-next": "^14.1.0",
     "mongodb": "^6.3.0",
-    "next": "^14.1.4",
+    "next": "^14.2.1",
```

Comment:
The patch seems to downgrade the version. With that said, it's more compatible with what we're building